client/post-editor/controller.js

```diff
@@ -131,10 +159,13 @@ module.exports = {
 			// We have everything we need to start loading the post for editing,
 			// so kick it off here to minimize time spent waiting for it to load
 			// in the view components
-			if ( postID ) {
+			if ( postID && ! copyId ) {
 				// TODO: REDUX - remove flux actions when whole post-editor is reduxified
 				actions.startEditingExisting( siteId, postID );
 				analytics.pageView.record( '/' + postType + '/:blogid/:postid', gaTitle + ' > Edit' );
+			} else if ( copyId ) {
+				startEditingPostCopy( siteId, copyId );
```

Comment:
Let's say this is some wishful thinking on my part.
I expect, once the post editor migration to Redux is complete, there'll be a single action to update all post attributes, making that function unnecessary.
But of course, right now it makes more sense creating a new Flux action.